src/cockpit/packages.py

```diff
@@ -106,19 +106,21 @@ def packages_loaded(self) -> None:
         """Called when the packages have been reloaded"""
 
 
-class BridgeConfig:
+class BridgeConfig(JsonObject):
     def __init__(self, value: JsonObject):
-        self.label = get_str(value, 'label', None)
+        super().__init__(value)
+
+        self.label = get_str(self, 'label', None)
 
-        self.privileged = get_bool(value, 'privileged', default=False)
-        self.match: JsonObject = get_dict(value, 'match', {})
+        self.privileged = get_bool(self, 'privileged', default=False)
+        self.match: JsonObject = get_dict(self, 'match', {})
         if not self.privileged and not self.match:
-            raise JsonError(value, 'must have match rules or be privileged')
+            raise JsonError(self, 'must have match rules or be privileged')
 
-        self.environ = get_strv(value, 'environ', ())
-        self.spawn = get_strv(value, 'spawn')
+        self.environ = get_strv(self, 'environ', ())
+        self.spawn = get_strv(self, 'spawn')
         if not self.spawn:
-            raise JsonError(value, 'spawn vector must be non-empty')
+            raise JsonError(self, 'spawn vector must be non-empty')
 
         self.name = self.label or self.spawn[0]
```

Comment:
Ah, because it's a `BridgeConfig` but never really used `__init__` which begs the question why this is even a thing. I must say the type hints are a tad bit confusing now as something which is a BridgeConfig is not actually "initialised" using `BridgeConfg(*foo)`